pepper-apis/src/test/java/org/broadinstitute/ddp/route/GetWorkflowRouteTest.java

```diff
@@ -152,28 +150,6 @@ public void testGet_fromActivityState_acceptsActivityCode() {
                 .body("next", equalTo(StateType.DONE.name()));
     }
 
-    @Test
-    public void testGet_fromActivityState_acceptsInstanceGuid() {
-        TransactionWrapper.useTxn(handle -> {
-            ActivityState current = new ActivityState(fromActivity.getActivityId());
-            ActivityState next = new ActivityState(nextActivity.getActivityId());
-            WorkflowTransition t1 = new WorkflowTransition(studyId, current, next, "true", 1);
-            insertTransitionsAndDeferCleanup(handle, t1);
-        });
-
-        ActivityInstanceDto fromInstance = TransactionWrapper.withTxn(handle ->
-                insertNewInstanceAndDeferCleanup(handle, fromActivity.getActivityId()));
-
-        given().auth().oauth2(token)
-                .queryParam(QueryParam.FROM, StateType.ACTIVITY)
-                .queryParam(QueryParam.INSTANCE_GUID, fromInstance.getGuid())
-                .when().get(url).then().assertThat()
-                .statusCode(200).contentType(ContentType.JSON)
-                .body("next", equalTo(StateType.ACTIVITY.name()))
-                .body("activityCode", equalTo(nextActivity.getActivityCode()))
-                .body("instanceGuid", is(nullValue()));
-    }
-
```

Comment:
This test checks if the route returns NULL `instanceGuid` for the "next" activity with a missing instance, it doesn't work this way now that we create a missing activity instance.

Reply:
We should keep this test. The point of the test is to check that it accepts instanceGuid in query params (as the test name suggests) not that the route returns null for instanceGuid. We can change it to check `is(notNullValue())`.

Reply:
Ok, thanks, didn't realize that. Put it back.